ext/node/resolution.rs

```diff
@@ -1078,14 +1078,16 @@ impl NodeResolver {
     url: &ModuleSpecifier,
   ) -> Result<PathBuf, AnyError> {
     let file_path = url.to_file_path().unwrap();
-    let mut current_dir = file_path.parent().unwrap();
+    let current_dir = self.fs.canonicalize(file_path.parent().unwrap())?;
+    let mut current_dir = current_dir.as_path();
     let package_json_path = current_dir.join("package.json");
     if self.fs.exists(&package_json_path) {
       return Ok(package_json_path);
     }
     let root_pkg_folder = self
       .npm_resolver
       .resolve_package_folder_from_path(&url.to_file_path().unwrap())?;
+    let root_pkg_folder = self.fs.canonicalize(&root_pkg_folder)?;
```

Comment:
I think we can remove this and then just use `current_dir` in `resolve_package_folder_from_path`. That said, doesn't `resolve_package_folder_from_path` already canonicalize?